setup.py

```diff
@@ -95,7 +95,7 @@ def get_data_files(name, version, fullname):  # pylint: disable=R0912
     systemd_dir_path = osutil.get_systemd_unit_file_install_path()
     agent_bin_path = osutil.get_agent_bin_path()
 
-    if name in ('redhat', 'centos', 'almalinux', 'cloudlinux'):  # pylint: disable=R1714
+    if name in ('redhat', 'centos', 'almalinux', 'cloudlinux', 'rocky'):  # pylint: disable=R1714
```

Comment:
Can you please remove this suppression, I don't think its needed anymore 

```suggestion
    if name in ('redhat', 'centos', 'almalinux', 'cloudlinux', 'rocky'):
```

Reply:
I removed the pylint portion on this line for you.